sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -11,4 +11,7 @@ export class ApplicationService {
   public async getNOA(applicationId: number): Promise<any> {
     return ApiClient.Application.getNOA(applicationId);
   }
+  public async confirmationOfAssessment(applicationId: number): Promise<void> {
```

Comment:
Can we have this method renamed to confirmAssessment? It seems to be more aligned with the others and with the API call.